prov/rxm/src/rxm.h

```diff
@@ -305,6 +307,9 @@ struct rxm_ep {
 	int			msg_cq_fd;
 	struct fid_ep 		*srx_ctx;
 	size_t 			comp_per_progress;
+	int			rx_buf_cnt;
+	int			max_rx_buf_cnt;
+	struct dlist_entry	rx_buf_list;
```

Comment:
In case the underlying provider doesn't support shared receive context, these may need to be per MSG endpoint.